spsa.py

```
@@ -1,454 +1,422 @@
+                        name, val, lo, hi, step, a, c = row
+                    else:
+                        raise ValueError(f"Bad row: {row}")
+                    params[name] = {
+                        "value": float(val),
+                        "min": float(lo),
+                        "max": float(hi),
+                        "step": float(step),
+                        "a": float(a),
+                        "c": float(c),
+                    }
+                except Exception as e:print(e, row)
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)